### ISSUE TITLE
RTD uses wrong version requirements

### ISSUE THREAD

**Issue by MatthieuSchaller** [SWIFTSIM/swiftsimio#166]
The online docs gives outdated python version requirements for the code. There might also be other things that are outdated there , for instance dependence versions.

**Comment by JBorrow** [comment]
Yeah I think this was related to the changed initialisation. Looking into it now.

**Comment by JBorrow** [comment]
Oops, I was fixing the actual docs build and not the contents, re-opening...
